docs/contribute.rst

```diff
@@ -702,7 +702,7 @@ make clean
 Release
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``tmt`` project is released monthly. If there are urgent
+The ``tmt`` project is released biweekly. If there are urgent
```

Comment:
Was this a conscious decision or accepting the reality of shorter release windows that started before Christmas?

Reply:
well, that is the current state isn't it?

Reply:
@psss WDYT? should I remove this change?

Reply:
It is the current state, the question is whether we decided this is what we want, or whether we are still shell-shocked by Christmas or whatever else and unable to return to the monthly release schedule. If we decided and agreed on the change, sure, let's update the line. If we are dragged by the reality and we do not want biweekly schedule and it's just what is is right now, we should rather change the schedule.

Reply:
you know what, I will add it to hacking and just remove this change